### PR TITLE
Update installation notes in README

### PR DESCRIPTION
**Pull request by spaghetticode**
## Description

This PR includes some small README improvements on the installation process.

See also issue #137 

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,10 @@ $ solidus_starter_frontend
 
 The last command will copy all the necessary files.
 
+**IMPORTANT**: if you plan to use `solidus_auth_devise` you also need to patch
+`SolidusSupport.frontend_available?` so it returns `true`, otherwise authentication
```
Comment (blocknotes) [inline README.md:64]: This is already implemented [here](https://github.com/nebulab/solidus_starter_frontend/blob/master/lib/solidus_starter_frontend/solidus_support_extensions.rb#L6).

Comment (blocknotes) [reply README.md:64]: Did you still get issues with auth routes?

Comment (blocknotes) [reply README.md:64]: Nevermind. I saw your issue #137
 
Can you link it in PR description?

**Review by kennyadsl — APPROVED** [review]

**Comment by blocknotes** [comment]
If we go on with the fix PR https://github.com/nebulab/solidus_starter_frontend/pull/138 we could avoid this README changes.

**Comment by spaghetticode** [comment]
@blocknotes thanks for the fix in #138  🎉 
 I'm closing this one as no more needed.

**Pull request closed without merging**